configs/config.pcengines_apu2

```diff
@@ -28,13 +36,15 @@ CONFIG_EDK2_OPAL_PASSWORD=y
 CONFIG_EDK2_SETUP_PASSWORD=y
 CONFIG_EDK2_PERFORMANCE_MEASUREMENT_ENABLE=y
 CONFIG_EDK2_DASHARO_SYSTEM_FEATURES=y
+CONFIG_EDK2_DASHARO_SECURITY_OPTIONS=y
```

Comment:
Shouldn't this be dropped from all configs?

Reply:
We need this for boot media lock.

Reply:
What effect does it have without SPI lock?  Configuring WP state of the  flash chip so the user doesn't need to do it manually via `flashrom`?

Reply:
Configuring SRP and BP bits in firmware. Flashrom can of course disable SRP and BP bits if WP pin is not asserted, so maybe there is no need to expose this...